core/jazz_ui/src/app/pages/service-detail/service-detail.component.ts

```diff
@@ -122,9 +121,15 @@ export class ServiceDetailComponent implements OnInit {
         endpoints: service.endpoints,
         deployment_targets :  service.deployment_targets[service.type].S || service.deployment_targets[service.type],
         is_public_endpoint: service.is_public_endpoint,
-        created_by: service.created_by,
-        accountID: service.deployment_accounts[0].accountId,
-        region: service.deployment_accounts[0].region
+        created_by: service.created_by
+      }
+      if(service.deployment_accounts){
+        returnObject['accountID'] = service.deployment_accounts[0].accountId,
+        returnObject['region'] = service.deployment_accounts[0].region
+      } else {
+        // If no deployment_accounts is present, then fetch from default aws region and account, otherwise fetch from provider (aws/azure) specific values. We are not assigning any default azure values.
```

Comment:
Nope, this is not needed. Every service should have deployment_accounts with accountId & region. Remove the `else` block.

Reply:
It is actually for the core services which are created during stack creation. I checked in the azure as well as sls-app stack. The core services doesn't have the deployment_accounts field. And for azure, now we have a new field in service list page called platform and since core service are missing the required field, so when in UI I open the any core services UI breaks. That's why this change.

Reply:
Same as above - Please see https://github.com/tmobile/jazz/pull/1156#discussion_r326942625

Reply:
Done. Removed the if else block. Services are expected to have now deployment_accounts field